src/main/resources/org/jenkinsci/test/acceptance/docker/fixtures/GitListContainer/Dockerfile

```diff
@@ -0,0 +1,62 @@
+#
+# Starts a container with sshd, git, gitlist
+# and prepares for execution of gitplugin tests.
+#
+
+FROM ubuntu:22.04
+
+RUN mkdir -p /var/run/sshd
+
+ARG DEBIAN_FRONTEND=noninteractive
+ENV TZ=Etc/UTC
+
+# install SSHD, Git and zip
+RUN apt-get update && apt-get install -y \
+    openssh-server \
+    git \
+    zip \
+    tar \
+    curl \
+    nginx-full \
+    php8.1-fpm \
```

Comment:
php-fqm fails to start for unknown reason. Still investigating.

Reply:
The original dockerfile was based on Ubuntu 14, so I have done tons of changes to upgrade it to Ubuntu 22. U14 didn't fully work locally for me so that's why I started this U22 upgrade work.